python/rikai/spark/sql/codegen/mlflow_logger.py

```diff
@@ -0,0 +1,123 @@
+#  Copyright (c) 2021 Rikai Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Custom Mlflow model logger to make sure models have the right logging for
+Rikai SQL ML
+"""
+from typing import Any, Optional
+
+import mlflow
+
+from rikai.conf import (
+    CONF_MLFLOW_ARTIFACT_PATH,
+    CONF_MLFLOW_MODEL_FLAVOR,
+    CONF_MLFLOW_OUTPUT_SCHEMA,
+    CONF_MLFLOW_POST_PROCESSING,
+    CONF_MLFLOW_PRE_PROCESSING,
+    CONF_MLFLOW_SPEC_VERSION,
+    CONF_MLFLOW_TRACKING_URI,
```

Comment:
nit: these keys are supposed to be used in `mlflow logger`? a bit confused that they are in the same module with `rikai.config.option` as well as the same configure key namespace `rikai.*`.

dont need to change anything tho.

Reply:
i'll move them to mlflow_logger for now (they're not registered in rikai.config.options right now)